config.h

```diff
@@ -186,6 +186,11 @@ struct pantavisor_metadata {
 	int usrmeta_interval;
 };
 
+struct pantavisor_vm {
+	int oom_qos_besteffort;
```

Comment:
whould be nice to have these in pv_config_get_json and pv_config_print too

Reply:
sure lets add there....